packages/devtools_app/lib/src/screens/inspector_shared/inspector_controls.dart

```diff
@@ -23,7 +23,7 @@ class InspectorControls extends StatelessWidget {
   final v2.InspectorController? controller;
 
   static const minScreenWidthForTextBeforeTruncating = 800.0;
-  static const minScreenWidthForTextBeforeScaling = 550.0;
```

Comment:
Can we revert the change to this file? I think this is for the left aligned inspector controls which we didn't need to adjust.

Reply:
Yes, done!